client_integration_test.go

```diff
@@ -1434,6 +1436,49 @@ func clientReadDeadlock(t *testing.T, N int, badfunc func(*File)) {
 	badfunc(r)
 }
 
+func TestClientSyncGo(t *testing.T) {
+	if !*testServerImpl {
+		t.Skipf("skipping without -testserver")
+	}
+	err := testClientSync(t)
+
+	// Since Server does not support the fsync extension, we can only
+	// check that we get the right error.
+	require.NotNil(t, err)
```

Comment:
Oof, I didn’t check the usage of `require` and `assert` as closely as I should have. :woman_facepalming: